src/main/kotlin/ai/digital/integration/server/deploy/tasks/server/DownloadAndExtractDbUnitDataDistTask.kt

```diff
@@ -6,7 +6,7 @@ import ai.digital.integration.server.deploy.internals.DeployExtensionUtil
 import ai.digital.integration.server.common.util.IntegrationServerUtil
 import org.gradle.api.tasks.Copy
 
-open class DownloadAndExtractDbUnitDataDistTask : Copy() {
+abstract class DownloadAndExtractDbUnitDataDistTask : Copy() {
```

Comment:
What is the reason this class is now abstract?

Reply:
<img width="1044" height="406" alt="image" src="https://github.com/user-attachments/assets/424e0a5d-65b0-4e75-8f83-542190003b81" />

due to the error :

Class 'DownloadAndExtractServerDistTask' is not abstract and does not implement abstract base class members:
fun getInstantiator(): Instantiator
fun getFileSystem(): FileSystem
fun getFileResolver(): FileResolver
fun getFileLookup(): FileLookup
fun getDirectoryFileTreeFactory(): DirectoryFileTreeFactory
fun getDocumentationRegistry(): DocumentationRegistry
fun getObjectFactory(): ObjectFactory
fun getPropertyFactory(): PropertyFactory

else needed to implement the base class members.

Reply:
<img width="799" height="736" alt="image" src="https://github.com/user-attachments/assets/ae71aa00-52b5-4341-b1cd-471d542ce4e5" />

Reply:
changes, updated to defaultTask instead of copy. fixed